modules/admin_manual/pages/configuration/files/previews_configuration.adoc

```diff
@@ -4,7 +4,7 @@
 == Introduction
 
 The ownCloud thumbnail system generates previews of files for all
-ownCloud apps that display files, such as Files and Gallery.
+ownCloud apps that display files, such as Files.
```

Comment:
This sounds so strange, the diffrence ist just lower/capital letter of (f)iles.
What about: `as the Files app` ?